cdc/processor.go

```diff
@@ -194,7 +194,7 @@ func newProcessor(
 		session:       session,
 		sink:          sink,
 		ddlPuller:     ddlPuller,
-		mounter:       entry.NewMounter(schemaStorage, changefeed.Config.Mounter.WorkerNum),
+		mounter:       entry.NewMounter(schemaStorage, changefeed.Config.Mounter.WorkerNum, changefeed.Config.Sink.OldValue),
```

Comment:
If we upgrade cdc server, the `changefeed.Config.Sink.OldValue` from changefeed info is nil. Recommend to add some compatibility code in the `VerifyAndFix` function of `ChangeFeedInfo`

Reply:
Oh, don't need this fix, the default value is `false`